src/vue/components/popup.vue

```diff
@@ -9,7 +9,7 @@
   >
     <div class="alpheios-popup__header">
       <div class="alpheios-popup__logo">
-        <logo-icon class="alpheios-logo-on-dark"/>
+        <logo-icon class="alpheios-logo-on-dark"></logo-icon>
```

Comment:
@kirlat , only interested - we were talking at the previous meeting that it was a good practice and recomended style - to create custom vue components without slot as a self-closing tags. Why did you change it?